src/metabase/api/collection.clj

```diff
@@ -958,16 +958,17 @@
     (premium-features/assert-has-feature :official-collections (tru "Official Collections"))
     (api/check-superuser))
   ;; Now create the new Collection :)
-  (first
-    (t2/insert-returning-instances!
-      Collection
-      (merge
-        {:name        name
-         :description description
-         :authority_level authority_level
-         :namespace   namespace}
-        (when parent_id
-          {:location (collection/children-location (t2/select-one [Collection :location :id] :id parent_id))})))))
+  (u/prog1 (first
+            (t2/insert-returning-instances!
```

Comment:
can this just use `insert-returning-instance!`?

Reply:
Woah, that's handy. Thanks for pointing that out. Fixed